static/styles/_base/_forms.scss

```diff
@@ -90,7 +90,24 @@ input[type="file"] {
 select {
   width: auto;
   max-width: 100%;
-  background: #fff;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  background-image: url('../img/angle-down.png'); // Fallback incase no svg background
+  background-image: url('../img/angle-down.svg');
+  background-repeat: no-repeat;
+  background-position: 95% 50%;
+  background-size: 2rem;
```

Comment:
Does this need a px fallback? Or maybe not because background-size isn't always implemented?